src/main/java/nextstep/optional/User.java

```diff
@@ -33,7 +35,9 @@ public static boolean ageIsInRange1(User user) {
     }
 
     public static boolean ageIsInRange2(User user) {
-        return false;
+        return Optional.ofNullable(user)
+                .filter((u) -> u.age != null && u.age >= 30 && u.age <= 45)
```

Comment:
```suggestion
                .filter((u) -> u.age != null)
                .filter((u) -> u.age >= 30)
                .filter((u) -> u.age <= 45)
```
이렇게 여러 개의 filter로 분리해서 작성해도 된답니다!